.changeset/fast-colts-cross.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/techdocs-common': minor
+'@backstage/plugin-techdocs': minor
+'@backstage/plugin-techdocs-backend': minor
```

Comment:
1. These should be patch changes. Since we are still v0.x.y, minor = breaking changes, patch = new features and bug fixes.
2. `@backstage/plugin-techdocs` can be omitted since we are not updated that package.